_episodes/02-numpy.md

```diff
@@ -89,8 +89,8 @@ that output is the data we just loaded.
 By default,
 only a few rows and columns are shown
 (with `...` to omit elements when displaying big arrays).
-To save space,
-Python displays numbers as `1.` instead of `1.0`
+To save space on the screen
+when printing a numpy array, numbers will be displayed as `1.` instead of `1.0`
 when there's nothing interesting after the decimal point.
```

Comment:
@coolernato I think the points you are clarifying are worthwhile ones to make. Thanks for proposing the change. A few things I am thinking:

1. We should capitalize "NumPy" here (based on other usage in the episode).
2. I wonder about using the word "printing" , since in this case we aren't using `print`.
3. Not sure about using the tense of "will be".

What do you think of the wording:
```
To save space on the screen when displaying a NumPy array, numbers are shown as `1.` instead of `1.0` when there's nothing interesting after the decimal point.
```

Reply:
@ldko Yes, I agree with your points and think your re-phrasing makes sense. I will update the request to use that phrasing.